Crowd Strike Falcon -  increase the default look-back (#38347)
demisto/content: Packs/CrowdStrikeFalcon/ReleaseNotes/2_1_9.md

#### Integrations

##### CrowdStrike Falcon

> - Fixed an issue where the default **look back** parameter was set to 1 minute instead of 2 minutes.

Reviewer: ```suggestion
- Updated the default **look back** parameter to 2 minutes instead of 1 minute.
```

Author: Added, thanks!
